sample-apps/donet-sample-app/Controllers/AppController.cs

```diff
@@ -30,6 +30,7 @@ public AppController()
             _metricEmitter.updateTotalHeapSizeMetric(rand.Next(100,500));
             _metricEmitter.updateTotalThreadSizeMetric(rand.Next(1,20));
             _metricEmitter.updateCpuUsageMetric(rand.Next(1,100));
+            // _metricEmitter.updateTotalHeapMetric(rand.Next(1,100));
```

Comment:
Why is this commented out? If it's not used we should remove it.

Reply:
Removed